src/commands/crashes/category.txt

```diff
@@ -0,0 +1 @@
+Commands related to Crashes
```

Comment:
For now the text should be: `Command to upload symbols to get symbolicated crash reports on Mobile Center`